tutorials/NeMo_Getting_Started.ipynb

```diff
@@ -0,0 +1,280 @@
+    "* Transcribe audio with (Russian) speech recognition model.\n",
+    "* Translate text with machine translation model.\n",
+    "* Generate audio with text-to-speech models."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "V72HXYuQ_p9a"
+   },
+   "source": [
+    "## Installation\n",
+    "NeMo can be installed via simple pip command.\n",
+    "This will take about 4 minutes.\n",
+    "\n",
+    "(The installation method below should work inside your new Conda environment or in an NVIDIA docker container.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "efDmTWf1_iYK"
+   },
+   "outputs": [],
+   "source": [
+    "BRANCH = 'r1.0.0rc1'\n",
+    "!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[all]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "EyJ5HiiPrPKA"
+   },
+   "source": [
+    "## Import all necessary packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "tdUqxeUEA8nw"
+   },
+   "outputs": [],
+   "source": [
+    "# Import NeMo and it's ASR, NLP and TTS collections\n",
+    "import nemo\n",
+    "# Import Speech Recognition collection\n",
+    "import nemo.collections.asr as nemo_asr\n",
+    "# Import Natural Language Processing colleciton\n",
+    "import nemo.collections.nlp as nemo_nlp\n",
+    "# Import Speech Synthesis collection\n",
+    "import nemo.collections.tts as nemo_tts\n",
+    "# We'll use this to listen to audio\n",
+    "import IPython"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "bt2EZyU3A1aq"
+   },
+   "source": [
+    "## Instantiate pre-trained NeMo models\n",
+    "\n",
+    "Every NeMo model has these methods:\n",
+    "\n",
+    "* ``list_available_models()`` - it will list all models currently available on NGC and their names.\n",
+    "\n",
+    "* ``from_pretrained(...)`` API downloads and initialized model directly from the NGC using model name.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "YNNHs5Xjr8ox"
+   },
+   "outputs": [],
+   "source": [
+    "# Here is an example of all CTC-based models:\n",
+    "nemo_asr.models.EncDecCTCModel.list_available_models()\n",
+    "# More ASR Models are available - see: nemo_asr.models.ASRModel.list_available_models()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "1h9nhICjA5Dk"
+   },
+   "outputs": [],
+   "source": [
+    "# Speech Recognition model - QuartzNet trained on Russian part of MCV 6.0\n",
+    "quartznet = nemo_asr.models.EncDecCTCModel.from_pretrained(model_name=\"stt_ru_quartznet15x5\").cuda()\n",
+    "# Neural Machine Translation model\n",
+    "nmt_model = nemo_nlp.models.MTEncDecModel.from_pretrained(model_name='nmt_ru_en_transformer6x6', strict=False).cuda()\n",
```

Comment:
Is the strict=False necessary?

Reply:
yes, I will undo it once NMT models are re-published.